src/authorize.js

```diff
@@ -37,16 +37,42 @@
       url += '&state=' + encodeURIComponent(redirectUri.substring(hashPos+1));
     }
     url += '&response_type=token';
+
+    if (global.cordova) {
+      return RemoteStorage.Authorize.openWindow(
+          url,
+          redirectUri,
+          'location=yes,clearsessioncache=yes,clearcache=yes'
+        )
+        .then(function(authResult) {
+          remoteStorage.remote.configure({
+            token: authResult.access_token
+          });
+
+          // TODO
+          // sync doesn't start until after reload
+          // possibly missing some initialization step?
+          global.location.reload();
+        })
+        .then(null, function(error) {
```

Comment:
Why add a second `then` with just the error handler instead of adding the handler to the first `then`?